src/types/markers.js

```diff
@@ -138,9 +140,10 @@ type GCMajorCompleted_Shared = {
   // This usually isn't present with the gecko profiler, but it's the same
   // as all of the slice markers themselves.
   slices_list?: GCSliceData[],
-};
+|};
 
-export type GCMajorCompleted = GCMajorCompleted_Shared & {
```

Comment:
This construct isn't well suited for exact objects: it's not possible to be 2 exact objects with different sets of properties at the same time.

Fortunately the object spread operator works perfectly well :)

Reply:
Thanks, I've wanted to change this. I feel `...` is more intuitive too.